src/main/java/com/google/cloud/tools/appengine/cloudsdk/internal/args/DevAppServerArgs.java

```diff
@@ -71,4 +72,12 @@
     }
     return Collections.emptyList();
   }
+
+  /**
+   * @return {@code [--name, key1=val1, --name, key2=val2, ...]} or {@code []} if
```

Comment:
You can have a method comment and omit the @return, but you can't do the converse. 
https://google.github.io/styleguide/javaguide.html#s7.2-summary-fragment